receiver/datadogrumreceiver/metadata.yaml

```
@@ -0,0 +1,14 @@
+type: datadogrum
+
+status:
+  class: receiver
+  stability:
+    development: [logs, traces, metrics]
+  distributions: [contrib]
```

```suggestion
  distributions: []
```
not currently in any distributions until it is in upstream